ansible/playbooks/playbook-deploy-portal.yml

```diff
@@ -1,10 +1,16 @@
 ---
-- hosts: newportalvms
+- hosts:
+  # - testportalvms
+  - newportalvms
   remote_user: ubuntu
   become: yes
   become_method: sudo
   connection: "ssh"
 
+  vars_files:
+    - group_vars/newportalvms/vars
+    - group_vars/newportalvms/vault
+
```

Comment:
Had to add `vars_files` manually to make it work - I remember having to run the playbook in a certain way to read vars automatically?

Reply:
I ran with `ansible-playbook -i buildnewportalvms.inventory playbook-deploy-portal.yml`

Reply:
You need to use a build-specific ansible inventory file (`buildnewportalvms.inventory`) and this needs to contain the following to build ubuntu22-test as a django portal
```
[newportalvms]
ubuntu22-test.genome.edu.au  ansible_connection=ssh ansible_user=ubuntu
```
Then run playbook with
`ansible-playbook playbook-deploy-portal.yml --inventory-file buildnewportalvms.inventory --limit ubuntu22-test.genome.edu.au`

Reply:
Awesome - this is the bit I was missing `--limit ubuntu22-test.genome.edu.au`

Reply:
Can you remove the `vars_files` section - it is not needed.